client/components/User.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  orders: PropTypes.arrayOf(
+    propTypes.shape({
+      id: PropTypes.number.isRequired,
+      updatedAt: PropTypes.string,
+      createdAt: PropTypes.string,
+    }),
+  ),
+};
+
+const propTypes = {
```

Comment:
Just wanted to point out that since propTypes is declared already, this will throw a syntax error. But I think you mentioned that this will be part of a different PR anyway, if so disregard!